discord/emoji.py

```diff
@@ -153,6 +155,37 @@ def guild(self):
         """:class:`Guild`: The guild this emoji belongs to."""
         return self._state._get_guild(self.guild_id)
 
+
+    def url_as(self,*,format=None,static_format="png"):
+        """Returns an :class:`Asset` for the emoji's url.
+
+        The format must be one of 'webp', 'jpeg', 'jpg', 'png' or 'gif', and
+        'gif' is only valid for animated emojis. 
+
+        Parameters
+        -----------
+        format: Optional[:class:`str`]
+            The format to attempt to convert the emoji to.
+            If the format is ``None``, then it is automatically
+            detected into either 'gif' or static_format depending on the
+            emoji being animated or not.
+        static_format: Optional[:class:`str`]
+            Format to attempt to convert only non-animated emoji's to.
```

Comment:
```suggestion
            The format to attempt to convert non-animated emojis to.
```
"only" is _probably_ unnecessary here.